packages/vuetify/src/components/VBtn/VBtn.sass

```diff
@@ -102,6 +102,14 @@
   opacity: 0
   transition: opacity .2s ease-in-out
 
+.v-btn__underlay
```

Comment:
?

Reply:
I remember it fixed some issue I was having. Although that might have been when I was experimenting with using the actual v-btn component as header instead of the sass mixin. I'll double-check.